fibonacci.dylan

```diff
@@ -20,7 +20,14 @@ define function fibo(M :: <integer>)
 end function fibo;
 
 begin
-  let arg = string-to-integer(element(application-arguments(), 0, default: "1"));
-  format-out("%d\n", fibo(arg));
-end;
+    let arg :: <integer> = string-to-integer(element(application-arguments(), 0, default: "1"));
+    let result :: <integer> = 0;
```

Comment:
These should be indented 2 spaces from the begin and should be at the same indentation level as the `profiling` call below.